drivers/bitget.js

```diff
@@ -0,0 +1,38 @@
+const Driver = require('../models/driver');
+const request = require('../lib/request');
+const Ticker = require('../models/ticker');
+const { parseToFloat } = require('../lib/utils');
+
+/**
+ * @memberof Driver
+ * @augments Driver
+ */
+class Bitget extends Driver {
+  /**
+   * @augments Driver.fetchTickers
+   * @returns {Promise.Array<Ticker>} Returns a promise of an array with tickers.
+   */
+  async fetchTickers() {
+    const { data: markets } = await request('https://api.bitget.com/api/spot/v1/public/products');
+    const { data: tickers } = await request('https://api.bitget.com/api/spot/v1/market/tickers');
+
+    return markets.flatMap((market) => {
+      const ticker = tickers.find((tickerItem) => tickerItem.symbol === market.symbolName);
+
+      // Not all markets might have tickers
+      if (!ticker) return [];
```

Comment:
In other drivers we return undefined. To keep some consistency and because we filter out undefined tickers, I suggest returning undefined here as well
```suggestion
      if (!ticker) return undefined;
```

Reply:
The empty arrays are filtered with the use of flatMap, which means we dont have to filter out undefined in a later part of the pipeline